### ISSUE TITLE
The results of reimplemented models

### ISSUE THREAD

**Issue by guangmingboy** [speedinghzl/pytorch-segmentation-toolbox#1]
I want to know the results of re-implemented models.

**Comment by speedinghzl** [comment]
PSPNet ~78.5 Deeplabv3 ~78.9 on cityscapes val dataset with single scale testing.
And we have uploaded the document to arxiv and it may take a little time to be public.

**Comment by speedinghzl** [comment]
The document is here https://weiyc.github.io/assets/pdf/toolbox.pdf

**Comment by lzrobots** [comment]
Are these results (pop 78.5, deeplabv3 78.9) based on the default cross-entropy loss?
So I assume the performance can be improved if OHEM loss is adopted (ohem flag=true)?
